api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -200,6 +205,9 @@ public MaterializeCohortResponse materializeCohort(@Nullable CohortReview cohort
         // There is no cohort review, so there are no annotations; return empty results.
         return response;
       }
+      if (statusFilter == null) {
+        statusFilter = REVIEWED_NOT_EXCLUDED;
+      }
```

Comment:
What if researchers don't care about statuses at all and only want to annotate participants. Seems like a researcher would want all annotations by default regardless of status?

Reply:
Changed to always include NOT_REVIEWED. This does mean FWIW that by default, you're going to get back NOT_REVIEWED rows (without annotations) for all the participants in the review you haven't even looked at... but you can always ask to not include them if you don't want them.

I'm still not including EXCLUDED by default, since it seems like researchers are saying "this person isn't in the cohort" by marking them that way. So IMHO our APIs and UIs should act as if those participants aren't in the cohort by default. I posted to #workbench-beta to doublecheck on this.